sass/base/space/_space.scss

```diff
@@ -5,46 +5,49 @@
   getThemeProperty(layoutSpace) variable is defined in _fw-variables.scss
   Used for consistent margins and gutters throughout the defined content
 */
+
+$default-base-space-properties:(
+  paddingclasses:(
+    default: getThemeProperty(layoutSpace),
```

Comment:
The only issue with my suggestion is that someone might want to apply padding on mobile, but not on desktop. To force the inclusion of the `fw-space` class for `fw-space-phone-default` to works means that the space will appear on desktop as well as on mobile. To workaround this, a "none" modifier could be added:
```scss
$default-base-space-properties:(
  paddingclasses:(
    none: 0,
```
That way an element with the classes: "fw-space none fw-space-phone-default" works the way that you would expect.